.github/workflows/wheels-dependencies.sh

```diff
@@ -137,14 +137,16 @@ function build_libavif {
         && CMAKE_POLICY_VERSION_MINIMUM=3.5 cmake \
             -DCMAKE_INSTALL_PREFIX=$BUILD_PREFIX \
             -DCMAKE_INSTALL_LIBDIR=$BUILD_PREFIX/lib \
+            -DCMAKE_INSTALL_NAME_DIR=$BUILD_PREFIX/lib \
+            -DCMAKE_MACOSX_RPATH=OFF \
             -DCMAKE_BUILD_TYPE=Release \
-            -DBUILD_SHARED_LIBS=OFF \
+            -DBUILD_SHARED_LIBS=ON \
```

Comment:
Did you mean to change `BUILD_SHARED_LIBS` to `ON`? It is still `OFF` in winbuild/build_prepare.py.

Reply:
The savings are modest, about 0.5 MB, but there isn't any cost to switching to a shared library.

Reply:
When you say 0.5mb, I presume that's per wheel. Which wheel(s) were you using to check that?